apps/portal/src/content/docs/components/file-additions-trigger.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: File Additions Trigger
+description: File Additions Trigger component documentation
+---
+
+The File Additions Trigger component provides a UI element for creating new files or uploading files. It uses a dropdown menu to offer these options.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<FileAdditionsTrigger
+  useTranslationStore={() => ({ t: (key, fallback) => fallback })}
+  pathNewFile="/new-file"
+  pathUploadFiles="/upload-files"
+/>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { FileAdditionsTrigger } from '@harnessio/ui/components'
+
```

Comment:
Remember to add the ellipsis to indicate that this isn't a fully working example.
```suggestion

// ...

```